src/chrome/content/rules/Fraunhofer.xml

```diff
@@ -3,21 +3,19 @@
 
 		- (www.)hhi *
 		- jp.hhi *
+		- izb (breaks design)
 
-	* Self-signed, mismatched, CN: svn.hhi.fraunhofer.de
+	* Self-signed, mismatched
 
 -->
-<ruleset name="Fraunhofer (partial)" default_off="mismatch, self-signed">
+<ruleset name="Fraunhofer (partial)">
 
-	<target host="scai.fraunhofer.de"/>
 	<target host="www.scai.fraunhofer.de"/>
-	<!--	mismatch	-->
-	<target host="izb.fraunhofer.de"/>
-	<target host="www.izb.fraunhofer.de"/>
+	<target host="fraunhofer.de"/>
+	<target host="www.fraunhofer.de"/>
 
 	<securecookie host="^(?:.*\.)?(?:izb|scai)\.fraunhofer\.de$" name=".*"/>
 
-	<rule from="^http://(?:www\.)?(izb|scai)\.fraunhofer\.de/"
```

Comment:
This rule used to rewrite the non-www version of these names to the www version. Have you confirmed both work?